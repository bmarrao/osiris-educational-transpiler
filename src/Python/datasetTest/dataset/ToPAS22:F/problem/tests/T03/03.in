10
1
1
1
-1
2
-1 -1
8
-1 -1 14 -1 16 -1 -1 -1
10
0 1 2 3 4 5 6 7 8 9
10
0 1 2 3 4 5 6 7 8 8
5
4 -1 -1 -1 -1
5
-1 -1 21 31 -1
5
-1 -1 21 32 -1
33
-1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1
