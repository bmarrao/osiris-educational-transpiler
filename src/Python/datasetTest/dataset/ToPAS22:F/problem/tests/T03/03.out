Possivel
Possivel
Possivel
Possivel
Possivel
Impossivel
Possivel
Possivel
Impossivel
Possivel
